dev-server/ui-playground/components/PlaygroundApp.js

```diff
@@ -0,0 +1,69 @@
+import { SvgIcon } from '@hypothesis/frontend-shared';
+
+import MenuPatterns from './patterns/MenuPatterns';
```

Comment:
Ah - I was thinking that we'd put the `patterns/` in a top level directory alongside the `components/` dir.